clang/test/Interpreter/execute.cpp

```diff
@@ -20,4 +20,6 @@ auto r2 = printf("S[f=%f, m=0x%llx]\n", s.f, reinterpret_cast<unsigned long long
 inline int foo() { return 42; }
 int r3 = foo();
 
+#include <memory>
```

Comment:
Why do we need this?

Reply:
This was to test (since `<memory>` transitively includes `<new>`).